### PR TITLE
Fixes for xenstore-srv stack and functions structure

### PR DESCRIPTION
**Pull request by firscity**

### CODE REVIEW

File: xenstore-srv/src/xenstore_srv.c [hunk 1]
```diff
@@ -1050,6 +986,91 @@ void xenstore_evt_thrd(void *p1, void *p2, void *p3)
 	cleanup_domain_watches(domain);
 }
 
+int start_domain_stored(struct xen_domain *domain)
+{
+	int rc = 0, err_ret;
```
Comment (LKomaryanskiy) [inline xenstore-srv/src/xenstore_srv.c:991]: (not your issue/nit): I think, as this function is global and can potentially be used from multiple places, we have to check input arguments, e.g:
```
if (!domain) {
	return -EINVAL;
}
```

File: xenstore-srv/src/xenstore_srv.c [hunk 2]
```diff
@@ -1050,6 +986,91 @@ void xenstore_evt_thrd(void *p1, void *p2, void *p3)
...
+	if (err_ret < 0) {
+		LOG_ERR("Failed to unmap domain#%u xenstore ring (rc=%d)",
+			domain->domid, err_ret);
+	}
+	return rc;
+}
+
+int stop_domain_stored(struct xen_domain *domain)
+{
+	int rc = 0, err = 0;
```
Comment (LKomaryanskiy) [inline xenstore-srv/src/xenstore_srv.c:1048]: (not your issue/nit): The same as start_domain_stored.

File: xenstore-srv/src/xenstore_srv.c (hunk 1 above)
Comment (firscity) [reply xenstore-srv/src/xenstore_srv.c:991]: Thanks, added in separate commit.

**Comment by lorc** [comment]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by LKomaryanskiy** [comment]
`Acked-by: Leonid Komarianskyi <leonid_komarianskyi@epam.com>`

**Pull request merged**